src/plugins/intel_gpu/src/kernel_selector/kernel_selector_params.h

```diff
@@ -169,6 +169,7 @@ class ParamsKey {
                     struct reorder_t {
                         uint32_t winograd : 1;
                         uint32_t rotate : 1;
+                        uint32_t surface_input : 1;
```

Comment:
surface output? I have a feeling that output_mem_type parameter is not actually supported, so maybe it can be removed?